sync/projectutil.py

```diff
@@ -39,8 +41,10 @@ def get(self, *subcommand, **opts):
         try:
             return subprocess.check_output(command, cwd=self.path, **opts)
         except subprocess.CalledProcessError as e:
-            newrelic.agent.record_exception(params={"mach_output": e.output})
-            raise e
+            newrelic.agent.record_exception(params={
+                "command": self.name,
+                "command_output": e.output})
+            raise AbortError("Problem running '%s' command" % self.name)
```

Comment:
I think on balance suggesting changing this to `AbortError` was wrong. There's actually no special handling for `AbortError` in the error reporting, and there's a place where we catch the `CalledProcessError` and have command-specific handling. 

In theory it would be better to have this handling for each instance of the command, as some cases might not be "exceptional" errors. But that's a lot of work and probably not worthwhile.

So I think reverting this back to just `raise` is actually the best trade-off. Sorry.